src/main/java/uk/gov/companieshouse/api/accounts/controller/CompanyAccountController.java

```diff
@@ -56,7 +56,7 @@ public ResponseEntity createCompanyAccount(@Valid @RequestBody CompanyAccount co
                 .map(result.getStatus(), result.getData(), result.getErrorData());
     }
 
-    @GetMapping
+    @GetMapping("/{companyAccountId}")
```

Comment:
Can you add this to the `RequestMapping` at the class level please?

Reply:
This can't be done as it'll be propagated onto the create, which is invalid